docs/UserGuide.md

```diff
@@ -513,8 +524,13 @@ in their address.
 * Only full words will be matched. <br>
   e.g. `Gardens,` will not match `Gardens` and `keychain` will not match `keychains`
 
+<<<<<<< HEAD
+* Orders matching at least one keyword will be displayed. <br>
```

Comment:
pls resolve these merge conflicts

Reply:
whoops thanks for catching that, its been resolved